### PR TITLE
[#406] Add Marionette Team section

### PR DESCRIPTION
**Pull request by trezy**
Adding a team section.

### CODE REVIEW

File: src/js/github.js [hunk 1]
```diff
@@ -10,3 +11,38 @@
...
+  })
+  .done(function(members) {
+    //jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+    /*jshint camelcase: false */
+    var i;
+    var memberCount = members.length;
+    var memberListEl = $('.member-list');
+
+    for (i = 0; i < memberCount; i++) {
+      var member = members[i];
```
Comment (samccone) [inline src/js/github.js]: no marionette :P

Comment (trezy) [inline src/js/github.js]: Ha, for some reason I didn't think we had Marionette in the page. That thought even struck me as odd.

Comment (samccone) [inline src/js/github.js]: screen shot 2015-10-26 at 9 58 34 am

Comment (trezy) [inline src/js/github.js]: Yup, I'll refactor to utilize it.

Comment (trezy) [inline src/js/github.js]: So we're not currently using Marionette. I'd say this isn't a blocker though I'm opening a separate issue to refactor the site to utilize it. Seems a little silly not to dogfood our own framework.

**Comment by jasonLaster** [comment]
:dancers:

**Pull request merged**